CONTRIBUTING.md

```diff
@@ -14,84 +12,67 @@ If you'd like to find out more about AutoEmulate, make sure to check out:
 
 1. **README**: For a high-level overview of the project, please refer to our README.
 2. **Documentation**: For more detailed information about the project, please refer to our [documentation](https://alan-turing-institute.github.io/autoemulate).
-3. **Project Roadmap**: Familiarise yourself with our direction and goals by checking out [the project's project board](https://github.com/orgs/alan-turing-institute/projects/185/views/4) in lieu of a formal product roadmap.
-
-## Get in touch
 
-The easiest way to get involved with the active development of AutoEmulate is to join our sprints. If you are looking to become part of the core development team in this way, reach out to Research Application Manager Kalle Westerling via email kwesterling@turing.ac.uk to request an invite.
-
-<!-- The easiest way to get involved with the active development of AutoEmulate is to join our regular community calls. The community calls are currently on a hiatus but if you are interested in participating in the forthcoming community calls, which will start in 2024, you should join our Slack workspace, where conversation about when to hold the community calls in the future will take place. -->
+## How to Contribute
 
-<!--
-**Slack Workspace**: Join our [AutoEmulate Slack channel](<LINK TO SIGN-UP OR TO THE SLACK TEAM>) for discussions, queries, and community interactions. Send us an email at kwesterling@turing.ac.uk to request an invite.
--->
+This section is intended to provide a high level overview of how to contribute to AutoEmulate if you don't have a lot of experience in contributing to open source projects. For more detailed information, please also refer to the docs for:
 
-**Email**: If you prefer a formal communication method or have specific concerns, please reach us via lead Research Software Engineer Martin Stoffel, mstoffel@turing.ac.uk.
+* [contributing emulators](contributing-emulators.md)
+* [contributing to the docs](contributing-docs.md)
 
-## How to Contribute
 
 We welcome contributions of all kinds, be it code, documentation, or community engagement. We encourage you to read through the following sections to learn more about how you can contribute to the package.
 
-We are always interested in adding more simulations or simulation input/output datasets from any field (see https://github.com/alan-turing-institute/autoemulate/issues/4).
```

Comment:
Yes, I good question:

1) I think we shouldn't just add lots of simulations to the package at this point
2) Instead, maybe we should write another document outlining specifically what type of simulation/data we're looking for and keeping the datasets broad, but few per discipline
3) I'm even thinking it might be best to export all data into a seperate benchmarking package eventually

But I agree opening an issue and being clear about this would be good.

Reply:
okay yes I agree we need to spent some thought on what exactly we want here. And I really like the idea of making this specifically around the benchmarking! 

tbd

Reply:
I have created a new issue for this #270.